### DISCUSSION TITLE
Support PDF uploads

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3016] [Ideas]
`imgproxy` allows for conversion of PDFs to images, so add support for PDF uploads.

**Comment by ekzyis** [comment]
> PDF only? What about epubs?

-- https://stacker.news/items/429663

Since I don't see EPUB specifically mentioned in the improxy docs[^1], I think if we want to support EPUBs, we need to convert them to PDF first.

Another option is to implement native uploads. But this might be another ticket or an iteration on this ticket for PDFs with a lot of pages.

[^1]: https://docs.imgproxy.net/image_formats_support

**Comment by brymut** [comment]
Still working on this @Soxasora ?
